### DISCUSSION TITLE
windows 10 x86_64 install on m1 macbook air

### DISCUSSION THREAD

**Discussion by mjonsson1986** [utmapp/UTM#3173] [UTM Help (macOS)]
are it iven possible?
cdrom drive as ide. iso file
qcow2 drive as ide drive
set the iso as first boot entry
starting the vm lands in uefi shell
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration**
* UTM Version: 
* OS Version: 
* Intel or Apple Silicon? 

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
Did you press any key to install windows when prompted?

**Comment by mjonsson1986** [comment]
The enter key Skickades från E-post för Windows Från: osySkickat: den 10 oktober 2021 18:20Till: utmapp/UTMKopia: mjonsson1986; AuthorÄmne: Re: [utmapp/UTM] windows 10 x86_64 install on m1 macbook air (Discussion #3173) Did you press any key to install windows when prompted?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
